build.wake

```diff
@@ -13,100 +13,103 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 package build_wake
 from wake import _
 
 # Useful build variants (arguments to all)
 def toVariant = match _
     "default" = Pass (Pair "native-cpp14-release" "native-c11-release")
-    "static"  = Pass (Pair "native-cpp14-static"  "native-c11-static")
-    "debug"   = Pass (Pair "native-cpp14-debug"   "native-c11-debug")
-    "wasm"    = Pass (Pair "wasm-cpp14-release"   "wasm-c11-release")
+    "static" = Pass (Pair "native-cpp14-static" "native-c11-static")
+    "debug" = Pass (Pair "native-cpp14-debug" "native-c11-debug")
+    "wasm" = Pass (Pair "wasm-cpp14-release" "wasm-c11-release")
     s = Fail "Unknown build target ({s})".makeError
 
+
 export def build: List String => Result String Error = match _
     "tarball", Nil = tarball Unit | rmap (\_ "TARBALL")
     kind, Nil =
         require Pass variant = toVariant kind
         all variant | rmap (\_ "BUILD")
     _ = Fail "no target specified (try: build default/debug/tarball)".makeError
 
+
 export def install: List String => Result String Error = match _
     dest, kind, Nil = doInstall (in cwd dest) kind | rmap (\_ "INSTALL")
     dest, Nil = doInstall (in cwd dest) "default" | rmap (\_ "INSTALL")
     _ = Fail "no directory specified (try: install /opt/local)".makeError
 
+
 # Build all wake targets
 def targets =
-    buildWake, buildWakeBox, buildFuseDaemon, buildShim, buildBSP, buildLSP, buildJobCache, buildWakeFormat, Nil
+    buildWake,
+    buildWakeBox,
+    buildFuseDaemon,
+    buildShim,
+    buildBSP,
+    buildLSP,
+    buildJobCache,
+    buildWakeFormat,
+    Nil
+
 
 def all variant =
-    require Pass x =
-        map (_ variant) targets
-        | findFail
+    require Pass x = map (_ variant) targets | findFail
     Pass (flatten x)
 
+
 # Install wake into a target location
 def doInstall dest kind =
     require Pass variant = toVariant kind
-    require Pass datfiles =
-        sources "{here}/share" `.*`
+    require Pass datfiles = sources "{@here}/share" `.*`
     def binfilesResult = all variant
     def releaseBin exe = installAs "{dest}/{replace `\.[^.]*$` '' exe.getPathName}" exe
-    require Pass datinstall =
-        findFailFn (installIn dest ".") datfiles
-    require Pass readmeSource =
-        source "README.md"
-    require Pass readme =
-        installIn "{dest}/share/doc/wake" "." readmeSource
+    require Pass datinstall = findFailFn (installIn dest ".") datfiles
+    require Pass readmeSource = source "README.md"
+    require Pass readme = installIn "{dest}/share/doc/wake" "." readmeSource
```

Comment:
They would get erased. In fact, even in the case where we fix alignment for `match`'s `=` this would still not be aligned. These `require` statements aren't associated to each other so it would be very challenging to do collective formatting. Basically anything that could be the common parent would have to check "do I have a list of requires as children" and then use some magic to see if those requires should be "connected"